<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCalView</class>
 <widget class="QWidget" name="CameraCalView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="m_FormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_LeftCameraLabel">
         <property name="text">
          <string>left camera</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QmitkDataStorageComboBox" name="m_LeftCameraComboBox">
         <property name="toolTip">
          <string>choose the left camera image node.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_RightCameraLabel">
         <property name="text">
          <string>(optional) right camera</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_RightCameraComboBox">
         <property name="toolTip">
          <string>choose the right camera image node.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_TrackerMatrixLabel">
         <property name="text">
          <string>(optional) tracker matrix</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_TrackerMatrixComboBox">
         <property name="toolTip">
          <string>chose the tracker matrix</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="m_GrabButton">
         <property name="toolTip">
          <string>grab image</string>
         </property>
         <property name="text">
          <string>grab</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_CalibrateButton">
         <property name="toolTip">
          <string>perform calibration</string>
         </property>
         <property name="text">
          <string>calibrate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_SaveButton">
         <property name="toolTip">
          <string>save calibration</string>
         </property>
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="m_ImageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkDataStorageComboBoxWithSelectNone</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBoxWithSelectNone.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
